docs/content/get-started/get-started.md

```diff
@@ -1,8 +1,10 @@
 # Get started: Introduction
 
+<iframe width="100%" height="512" src="https://www.youtube.com/embed/6PNDBYtIimw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
I'm not sure if autoplay is a good idea

Reply:
Autoplay removed and related videos set to channel-only. Thanks!